packages/backend/src/controllers/connectors/logger.ts

```diff
@@ -14,59 +14,75 @@ class ConnectorLogger {
   private createContent(message: string, objectId: string): { [key: string]: string } {
     return {
       connector: objectId,
-      message
+      message,
     };
   }
 
   /**
-   * Prints and stores an info message
+   * Creates a new ConnectorLogItem and save it to the database.
+   *
+   * @private
+   * @param {{ [key: string]: any, type: ConnectorLogType }} content
+   * @return {Promise<void>} Return nothing
```

Comment:
```suggestion
   * @param content
   * @return Return nothing
```
Jsdoc type annotations are not specified for the majority of the other functions because of typescript. We should handle this consistently